application/controllers/Homepage.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Homepage extends Application
+{
+
+	/**
+	 * Index Page for this controller.
+	 */
+	public function index()
+	{
+		$this->data['pagebody'] = 'homepage';
```

Comment:
Tab indents in this whole file

Reply:
fixed. also found an atom plugin to prevent this going forward